classes/01class/exercises/c01-aws01/shahriar048/README.md

```diff
@@ -0,0 +1,330 @@
+# C01-AWS01
+
+## Commands Execution Output
+
+- Commands to create the first EC2 instance and any additional resource required:
+```
+$ `aws configure`
+
+Output:
+  AWS Access Key ID [None]: *
+  AWS Secret Access Key [None]: *
+  Default region name [None]: ap-southeast-2
+  Default output format [None]: 
+```
+
+```
+$ `aws ec2 create-key-pair --key-name MyKeyPair --query 'KeyMaterial' --output text > MyKeyPair.pem`
+$ `chmod 400 MyKeyPair.pem`
+$ `aws ec2 create-security-group --group-name my-sg --description "My security group" --vpc-id vpc-49e2eb2e`
+
+Output:
+    {
+        "GroupId": "sg-0d9198b56e119769d"
+    }
+```
+
+```
+$ `aws ec2 authorize-security-group-ingress --group-id sg-0d9198b56e119769d --protocol tcp --port 22 --cidr 0.0.0.0/0`
```

Comment:
By allowing this cidr `0.0.0.0/0`, anyone in the internet can access this instance.
Would there be more secure way to expose this instance?